tests/test_utils.py

```diff
@@ -1,5 +1,22 @@
 from __future__ import unicode_literals
-from nap.utils import make_url, is_string_like, handle_slash
+
+from nap.utils import make_url, is_string_like, handle_slash, normalize_url
+
+
+class TestNormalizeUrl(object):
+    def test_normalize_url(self):
+        url                     = "https://s9106.qa.cashstar.com/v1/catalogs/?merchant_code=DELL&promo=ALL&country=ALL&currency=ALL&context=BULK&context=BULKP"
```

Comment:
I think PEP8 shuns this kind of formatting. Search for "long_variable" on http://legacy.python.org/dev/peps/pep-0008/#pet-peeves